parmed/openmm/parameters.py

```diff
@@ -101,10 +107,20 @@ def from_parameterset(cls, params, copy=False):
         new_params.parametersets = params.parametersets
         new_params._combining_rule = params.combining_rule
         new_params.residues = params.residues
+        # Assign Lennard-Jones atom classes
+        ljtypes = dict()
+        counter = 0
+        for name, atom_type in iteritems(new_params.atom_types):
+            key = (atom_type.rmin, atom_type.epsilon, atom_type.rmin_14,
+                    atom_type.epsilon_14)
+            if key not in ljtypes:
+                ljtypes[key] = counter
```

Comment:
> Our primary concern is to translate the original parameter specification files (*.dat files for AMBER) into faithful representations of these parameter sets in ffxml files,

I think the primary concern should be to translate the original parameter specification files (\*.dat *and* \*.frcmod -- don't forget that all FFs are really combinations of these) into XML files that give equivalent energies.  Faithful representations are impossible where you need a `<script>` section, since no other program has an analogy to that.

> ideally in a way that maximizes our ability to mix-and-match sets of these files.

Good point.  My approach would not work here.  But just using types (and no classes) *would* ;).

That said, taking one of the atom type names should work nicely.  I'll probably just keep the first type corresponding each L-J parameter type as the class name.